MAINTAINERS.yml

```diff
@@ -4360,6 +4360,8 @@ West:
   maintainers:
     - KhiemNguyenT
     - andrzej-kaczmarek
+    - blauret
```

Comment:
can you please add a collaborator section and add collaborators in there. 4 maintainers for 1 area is a bit too much which makes triaging and approval a bit tricky.

Reply:
Done!